engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Builtins.java

```diff
@@ -174,8 +174,13 @@ private void registerBuiltinMethods(ModuleScope scope, EnsoLanguage language) {
         methods.entrySet().stream().forEach(entry -> {
           Type tpe = entry.getValue().isAutoRegister ? (!entry.getValue().isStatic() ? type : type.getEigentype()) : null;
           if (tpe != null) {
-            Optional<BuiltinFunction> fun = entry.getValue().toFunction(language);
+            LoadedBuiltinMethod value = entry.getValue();
+            Optional<BuiltinFunction> fun = value.toFunction(language, false);
             fun.ifPresent(f -> scope.registerMethod(tpe, entry.getKey(), f.getFunction()));
+            if (!value.isStatic()) {
+              Optional<BuiltinFunction> fun2 = value.toFunction(language, true);
+              fun2.ifPresent(f -> scope.registerMethod(tpe, entry.getKey() + "_static", f.getFunction()));
```

Comment:
Is usage of `"_static"` suffix safe? What if there are methods:
```
my = 42
my_static = 6 * 7
```
will not they accidentally override one another?

Reply:
Yes. But I will make it easier - since regular functions and static wrappers now share the same implementation I was able to ditch this fragment completely. And native image is still happy.